src/Comment.jsx

```diff
@@ -70,11 +68,6 @@ class _Comment extends React.Component {
     componentWillReceiveProps(newProps) {
         let accessTokenChanged = this.props.accessToken !== newProps.accessToken;
         let commentUriChanged = this.props.commentUri !== newProps.commentUri;
-        let localeChanged = this.props.locale !== newProps.locale;
-
-        if (localeChanged) {
-            newProps.i18n.changeLanguage(newProps.locale);
```

Comment:
Changing language is not really needed anymore because it is explicitly set (line 218) at translation time.